unittests/finalizer_update_tests.cpp

```diff
@@ -101,8 +101,8 @@ BOOST_AUTO_TEST_CASE(savanna_set_finalizer_multiple_test) { try {
    auto pubkeys2 = fin_keys.set_finalizer_policy(2u).pubkeys;
    t.produce_block();
    t.check_head_finalizer_policy(1u, pubkeys0); // new policy should only be active until after two 3-chains
```

Comment:
many comments in this file still mention `3-chains`. Some in other files as well.

Reply:
To be fixed by https://github.com/AntelopeIO/spring/issues/398